docs/commitstyle.md

```diff
@@ -1,46 +1,9 @@
 # Commit Style Guidelines for Kiwix-Android
 
 ### Message Structure
-Commit messages consist of three distinct parts, separated by a blank line: the title, an optional body/content, and an optional footer/metadata. The layout looks like this:
+Commit messages should be in the following format:
 
-type subject
-
-body
+    #<issue-number> A brief description of changes on one line
```

Comment:
What if the commit doesn't relate to a specific issue, or relates to more than one issue?

Reply:
Ideally it should always relate to an issue. If you are working on something that fixes multiple issues you should probably create a super issue that references the other issues?
Ironically in this pr I was lazy and did not create an issue...